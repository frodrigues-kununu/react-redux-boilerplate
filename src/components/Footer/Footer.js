import React from 'react';

const Footer = () => (
  <footer className="page-footer">
    Copyright @ me
  </footer>
);


export default Footer;
