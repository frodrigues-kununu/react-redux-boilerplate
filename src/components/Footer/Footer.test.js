import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJSON from 'enzyme-to-json';
import Footer from './Footer';

Enzyme.configure({ adapter: new Adapter() });

function setup() {
  const props = {};

  const enzymeWrapper = shallow(<Footer {...props} />);

  return {
    props,
    enzymeWrapper,
  };
}

//test wrapper and props
const { enzymeWrapper, props } = setup();

describe('<Footer />', () => {
  //snapshot testing
  test('should render correctly', () => {
    // const tree = renderer
    //   .create(<Header page="http://www.facebook.com"></Header>)
    //   .toJSON();

    expect(toJSON(enzymeWrapper)).toMatchSnapshot();
  });
});
