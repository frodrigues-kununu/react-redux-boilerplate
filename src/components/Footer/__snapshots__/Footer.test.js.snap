// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render correctly 1`] = `
<footer
  className="page-footer"
>
  Copyright @ me
</footer>
`;
